models/trackers.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+from typing import Optional
+from datetime import datetime
+
+
+class Tracker(BaseModel):
+    name: Optional[str]
+    url_address: str
+    created_at: Optional[datetime] = datetime.now()
+    is_offer: Optional[bool] = None
```

Comment:
I would remove this from the model, or put frequency instead. Even though we won't be using it now, we know that we will definitely need it in the future. We could set it optional and put 24 as the default value.